src/modules/location/index.ts

```diff
@@ -320,37 +320,53 @@ export class LocationModule {
    * Returns a random [ISO_3166-1](https://en.wikipedia.org/wiki/ISO_3166-1) country code.
    *
    * @param options The code to return or an options object. Defaults to `{}`.
-   * @param options.variant The variant to return. Can be either `'alpha-2'` (two-letter code)
-   * or `'alpha-3'` (three-letter code). Defaults to `'alpha-2'`.
+   * @param options.variant The variant to return. Can be one of:
+   * - `'alpha-2'` (two-letter code)
+   * - `'alpha-3'` (three-letter code)
+   * - `'numeric'` (numeric code)
+   * Defaults to `'alpha-2'`.
```

Comment:
```suggestion
   * @param options.variant The variant to return. Can be one of:
   *
   * - `'alpha-2'` (two-letter code)
   * - `'alpha-3'` (three-letter code)
   * - `'numeric'` (numeric code)
   *
   * Defaults to `'alpha-2'`.
```

Otherwise it will be displayed as:

![grafik](https://github.com/faker-js/faker/assets/1579362/e95ed754-1d43-4ad3-8a52-b7066a7f1270)